quantmsio/core/mztab.py

```
@@ -2,539 +1,263 @@
+import codecs
+import os
+import pandas as pd
+import re
```

Ruff

`re` imported but unused

Remove unused import: `re`

(F401)